### PR TITLE
Fixes #9496 - Adds popover help to IPAM field in Subnet form

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by dLobatog** [comment]
[test]

**Comment by ares** [comment]
How can I get new cops warning in code I did not touch? Do we have issue in develop branch already?

File: app/views/subnets/_fields.html.erb [hunk 1]
```diff
@@ -5,7 +5,16 @@
 <%= text_f f, :dns_primary, :help_inline => _("Optional: Primary DNS for this subnet") %>
 <%= text_f f, :dns_secondary, :help_inline => _("Optional: Secondary DNS for this subnet") %>
 <%= selectable_f f, :ipam, Subnet.ipam_modes_with_translations, {}, :data => {'disable-auto-suggest-on' => Subnet::IPAM_MODES[:none]},
-                 :label => _('IPAM'), :help_block => _('IP Address auto suggestion mode for this subnet, DHCP works with DHCP proxy only') %>
+                 :label => _('IPAM'),
+                 :help_inline => popover(
+                         _("IP Address Management"),
+                         _("You can select one of three possible IPAM modes:<br/>" +
+                                   "<ul><li><strong>DHCP</strong> - will manage the IP on DHCP through assigned DHCP proxy, auto-suggested IPs come from DHCP</li>" +
```
Comment (dLobatog) [inline app/views/subnets/_fields.html.erb:12]: This text might be a bit nightmarish to translate with so many labels - but I can't think of anything better now, @domcleal maybe you want to follow up this PR with something?

**Comment by dLobatog** [comment]
Merged as faf2af47aa01d3ef1bcb936e185c94703ea6ae3f, thanks @ares !

**Pull request closed without merging**